packages/graph-editor/src/index.tsx

```diff
@@ -1,5 +1,7 @@
-export * from './utils/index.ts';
-export * from './types/index.ts';
-export * from './redux/index.tsx';
-export * from './editor/index.tsx';
+export * from './components';
```

Comment:
Are these working without throwing an warnings/errors? The resolution type we were using before  necessitated that we used full paths

Reply:
yes, working